tests/unit_tests/test_outputtransform.py

```diff
@@ -62,6 +75,11 @@ def test_output_transform():
         frag_df.to_parquet(os.path.join(raw_folder, "frag.parquet"), index=False)
         psm_df.to_parquet(os.path.join(raw_folder, "psm.parquet"), index=False)
 
+        optimization_manager = manager.OptimizationManager(
+            config, os.path.join(raw_folder, "optimization_manager.pkl")
```

Comment:
is this pkl available in the tests?

Reply:
Yes, it's saved each time optimization_manager.fit() is called.